invokeai/frontend/web/src/features/parameters/components/Prompts/ViewModePrompt.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Flex, Icon, Text, Tooltip } from '@invoke-ai/ui-library';
+import { useAppDispatch } from 'app/store/storeHooks';
+import { viewModeChanged } from 'features/stylePresets/store/stylePresetSlice';
+import { getViewModeChunks } from 'features/stylePresets/util/getViewModeChunks';
+import { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { PiEyeBold } from 'react-icons/pi';
+
+export const ViewModePrompt = ({ presetPrompt, prompt }: { presetPrompt: string; prompt: string }) => {
+  const dispatch = useAppDispatch();
+  const { t } = useTranslation();
+
+  const presetChunks = useMemo(() => {
+    return getViewModeChunks(prompt, presetPrompt);
+  }, [presetPrompt, prompt]);
+
+  const handleExitViewMode = useCallback(() => {
+    dispatch(viewModeChanged(false));
+  }, [dispatch]);
+
+  return (
+    <Box position="absolute" top={0} bottom={0} left={0} right={0} zIndex={1} layerStyle="second" borderRadius="base">
```

Comment:
`z-index` is another CSS property to avoid if possible. CSS creates [stacking contexts](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) for positioning things in the z-axis, without the need to set `z-index`.

Instead of setting `z-index` here, in the parent component, we can move `<ViewModePrompt />` to be the last child of the parent `<Box />`. Same result visually, but without the layout interactions introduced by explicitly setting `z-index`.